condense_sdrf.pl

```diff
@@ -308,6 +319,25 @@ sub copy_idf_from_ae {
     }
 }
 
+sub copy_sdrf_to_output_dir {
+    my ( $investigation, $output_dir, $idf_abs_path ) = @_;
+    foreach my $sdrf ( @{ $investigation->get_sdrfs() } ) {
+        my $filename = $sdrf->get_uri()->file();
+        if( !File::Spec->file_name_is_absolute( $filename ) ) {
```

Comment:
This mostly assumes that SDRFs are in the same directory as the IDF, which is sound for Atlas, but this is not very defensive for more complex cases.